guides/typography.md

```diff
@@ -33,137 +36,72 @@ requests][font-inlining].
-  $line-height: 1,
-  $letter-spacing: normal,
-);
-```
-
-## Typography config
-
-A **typography config** is a collection of all typography levels. Angular Material represents this
-config as a Sass map. This map contains the styles for each level, keyed by name. You can create
-a typography config with the `define-typography-config` Sass function. Every parameter for
-`define-typography-config` is optional; the styles for a level will default to Material Design's
-baseline if unspecified.
-
-```scss
-@use '@angular/material' as mat;
+The following aspects of your app's typography can be customized via the `typography` property of
+the theme configuration object.
 
-$my-custom-typography-config: mat.define-typography-config(
-  $headline-1: mat.define-typography-level(112px, 112px, 300, $letter-spacing: -0.05em),
-  $headline-2: mat.define-typography-level(56px, 56px, 400, $letter-spacing: -0.02em),
-  $headline-3: mat.define-typography-level(45px, 48px, 400, $letter-spacing: -0.005em),
-  $headline-4: mat.define-typography-level(34px, 40px, 400),
-  $headline-5: mat.define-typography-level(24px, 32px, 400),
-  // ...
-);
-```
+| Typography Property | Description                                                          |
+| ------------------- | -------------------------------------------------------------------- |
+| `plain-family`      | [Optional] The font family to use for plain text, such as body text. |
+| `brand-family`      | [Optional] The font family to use for brand text, such as headlines. |
+| `bold-weight`       | [Optional] The font weight to use for bold text.                     |
+| `medium-weight`     | [Optional] The font weight to use for medium text.                   |
+| `regular-weight`    | [Optional] The font weight to use for regular text.                  |
 
-### Typography configs and theming
+These are used to generate the styles for the different typescale levels.
 
-You can provide a typography config when defining a theme to customize typographic styles. See the [theming guide][theming-system] for details on custom themes.
+## Type scale levels
 
-The following example shows a typical theme definition and a "kids theme" that only applies when
-the `".kids-theme"` CSS class is present. You can [see the theming guide for more guidance on
-defining multiple themes](https://material.angular.io/guide/theming#defining-multiple-themes).
+A **type scale** is a selection of font styles that can be used across an app.
+They’re assigned based on use (such as display or headline), and grouped more
+broadly into categories based on scale (such as large or small). For more
+information, see the [M3 typography spec](https://m3.material.io/styles/typography/type-scale-tokens).
 
-```scss
-@use '@angular/material' as mat;
+There are `large`, `medium`, and `small` variations for the following type roles:
+- **Display**: Styles are reserved for short, important text or numerals. They work best on large screens.
```

Comment:
The rest of the bullets repeat the name of the role (`**Title**: Titles ...`). We should be consistent, either always repeat or never repeat